### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by zaro0508**
This PR builds on top of the work done by @dboitnot in PR #2
to implement the sceptre-json-resolver. 

Co-authored by @dboitnot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+Take some json object serialize it to a string then pass it to a parameter:
+```yaml
+parameters:
+   myparam: !to_json [{"key": "value"}]
+```
+
+Take a string deserialize it to a json object then pass it to a parameter:
+```yaml
+sceptre_user_data:
+  myparam: !from_json ['{"key": "value"}']
```
Comment (jfalkenstein) [inline README.md]: I think you'll need to clarify somewhere that it expects a 1-item list parameter. I know it's reflected in your examples, but it's not _explained_. That's weird enough that I feel like it probably ought to receive some mention.

File: resolver/from_json.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                arg_count = len(arg)
+                if arg_count != 1:
+                    raise SceptreException(
+                        f"!from_json expects exactly one argument, got {arg_count}"
+                    )
+                return json.dumps(arg[0])
+
+            raise SceptreException(
+                f"!from_json expects a string argument, got {type(arg)}"
+            )
```
Comment (jfalkenstein) [inline resolver/from_json.py]: It actually expects a list argument, not a string.

**Comment by dboitnot** [comment]
I closed PR #2 in favor of this one.

File: resolver/from_json.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            if isinstance(arg, list):
+                arg_count = len(arg)
+                if arg_count != 1:
+                    raise SceptreException(
+                        f"!from_json expects exactly one argument, got {arg_count}"
+                    )
+                return json.dumps(arg[0])
+
+            raise SceptreException(
+                f"!from_json expects a string argument, got {type(arg)}"
```
Comment (dboitnot) [inline resolver/from_json.py]:
```suggestion
                f"!from_json expects a single-item list argument, got {type(arg)}"
```

**Review by dboitnot — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zaro0508) [reply README.md]: done

File: pyproject.toml [hunk 4]
```diff
@@ -30,7 +31,7 @@ pre-commit = "^3.2.1"
 tox = "^4.0.0"
 pytest = "^7.2.2"
 pytest-cov = "^4.0.0"
-sceptre = ">3.2"
+sceptre = ">=4.0.2"
```
Comment (zaro0508) [inline pyproject.toml]: need to make a sceptre v4.1.0 release then update this to `sceptre = ">=4.1.0"`

Comment (zaro0508) [reply pyproject.toml]: done

**Pull request merged**